tools/circle_plus_gen/schema/circle_traininfo_generated.py

```diff
@@ -1,3 +1,4 @@
+# generated based on commit(b3ff53668e165843d99a108d16ab14bc2cfca669)
```

Comment:
commit id points to here -https://github.com/Samsung/ONE/commit/b3ff53668e165843d99a108d16ab14bc2cfca669. 
It is the last commit of master branch when I created this PR.